torch/_decomp/decompositions.py

```diff
@@ -2636,6 +2686,43 @@ def one_layer_lstm_data(inp, hidden, params, has_biases, batch_sizes, reverse=Fa
     return out, hidden_out
 
 
+def use_mkldnn(input, hx, params):
```

Comment:
add a docstring to summarize the condition here? Also, it is specific to lstm, is the name `use_mkldnn` too general? Or maybe move the functions as local to `lstm_impl`?

Reply:
Docstring has been updated. Moved `use_mkldnn` to be a local function.